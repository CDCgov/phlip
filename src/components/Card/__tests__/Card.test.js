import React from 'react'
import { shallow } from 'enzyme'
import { Card } from '../index'

const props = { children: <div>I am card children.</div>, style: {} }

describe('Card component', () => {
  test('should render correctly', () => {
    expect(shallow(<Card {...props} />)).toMatchSnapshot()
  })
})