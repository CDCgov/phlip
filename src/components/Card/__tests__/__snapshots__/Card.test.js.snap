// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component should render correctly 1`] = `
<withStyles(Paper)
  style={
    Object {
      "flex": "1",
    }
  }
>
  <div>
    I am card children.
  </div>
</withStyles(Paper)>
`;
